kolibri/plugins/pwa/templates/manifest.json

```diff
@@ -0,0 +1,37 @@
+{% load i18n core_tags webpack_tags content_tags cache %}
```

Comment:
I don't think you are using `core_tags webpack_tags content_tags cache` here, so you could probably just leave this as `i18n`.